toxcore/tox.c

```diff
@@ -372,7 +372,6 @@ static int tox_load(Tox *tox, const uint8_t *data, uint32_t length)
                       length - cookie_len, STATE_COOKIE_TYPE);
 }
 
-
 Tox *tox_new(const struct Tox_Options *options, Tox_Err_New *error)
```

Comment:
Should we put something in here to abort when compiled in fuzzing mode?

Reply:
How would we fuzz the though? What do you want to achieve with that?

Reply:
Never mind, I thought the fuzz targets are set up with internals, not with a Tox instance.